src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Draw, Area {
```

Comment:
Don't use abstract classes to set behavior for classes. Consider using an interface if you need to define a contract without providing a base implementation, or a concrete class if you need to share common behavior.